### PR TITLE
Adding a test to detect duplicate OpenAPI schemas

### PR DESCRIPTION
**Pull request by andrewrlee**
Duplicate schemas do not render in the swagger docs - only one version will be displayed everywhere

A couple of duplicates were fixed a couple of weeks ago and this changes another and ignores an instance that affects display of the V1 APIs

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/adjudications/AdjudicationDetail.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
 
-@Schema(description = "Detail about an individual Adjudication")
+@Schema(name = "IndividualAnnotation", description = "Detail about an individual Adjudication")
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/hmpps/prison/api/model/adjudications/AdjudicationDetail.java:15]: should this be `IndividualAdjudication` instead?

**Review by petergphillips — APPROVED** [review]
LGTM
